docs/backplane/requirements/README.md

```diff
@@ -0,0 +1,30 @@
+# Backplane Requirements
```

Comment:
How does this align with #1587 ?

Reply:
https://github.com/openshift/managed-cluster-config/pull/1587 provides guidelines.  This is govered by SD-ADR-0138, not the guidelines.  Should we capture the ADR in the repo for full context of the workflow?